client/src/data/types/donorHomepageConfig.ts

```diff
@@ -30,12 +30,6 @@ export interface DonorHomepageConfig {
         testimonials: Array<Testimonial>;
     };
     statistics: Array<Statistic>;
-    banner: {
-        header: string;
-        description: string;
-        imagePaths: Array<string>;
-        interval: number;
-    };
```

Comment:
how come banner isn't needed?

Reply:
oh just cause we don't edit it on this page